Services/ActiveRecord/Fields/class.arFieldList.php

```diff
@@ -16,7 +16,6 @@ class arFieldList {
 	const FIELDTYPE = 'fieldtype';
 	const LENGTH = 'length';
 	const SEQUENCE = 'sequence';
-	const INDEX = 'index';
```

Comment:
Indices are handled in 
Services/ActiveRecord/Connector/class.arConnectorDB.php:70:
public function updateIndices(ActiveRecord $ar);

Please restore the removal of INDEX